tools/bioimaging/.shed.yml

```diff
@@ -0,0 +1,11 @@
+name: bioimage_inference
```

Comment:
@anuprulez do you plan to add a second tool to this suite? If so this should be a suite based shed.yml file.

If we plan to add the tensorflow functions into this one tool, it can stay like it is.

Reply:
I think the TensorFlow model rendering should also be included in this tool. But, adding that feature might need some time. It should be one tool.